Library/brew.rb

```diff
@@ -94,14 +94,8 @@ def require?(path)
     exit Homebrew.failed? ? 1 : 0
   else
     require "tap"
-    possible_tap = case cmd
-    when "brewdle", "brewdler", "bundle", "bundler"
-      Tap.fetch("Homebrew", "bundle")
-    when "cask"
-      Tap.fetch("caskroom", "cask")
-    when "services"
-      Tap.fetch("Homebrew", "services")
-    end
+    possible_tap = OFFICIAL_CMD_TAPS.find { |_, repo| repo == cmd }
```

Comment:
> Are you intentionally dropping support for auto-tapping when brew bundle is invoked with the alternative commands brewdle, brewdler, and bundler? (I'm not opposing this change, just making sure.)

Yes. PR incoming for homebrew-bundle to do that.



> I am however opposite to only check cmd with tap name, which it's a bad assumption for future commands or taps.

I don't think it's a bad assumption but we can correct it when it's no longer correct as it's an implementation detail.